mne/annotations.py

```diff
@@ -24,6 +24,35 @@
 from .io.tag import read_tag
 
 
+def _check_ods(onset, duration, description):
```

Comment:
Long names help readability in terms of English readability, but can hurt in terms of code readability when we then go to make things < 80 char. So for private things like this I prefer to keep things short. I don't like `_check_annotation_attributes` because it's only checking some of the attributes, not all as implied by the name. I'll change it to `_o_d_s` to make it clearer they are separate words